### PR TITLE
Added indicators, updated connector, changed parameters.

### PR DESCRIPTION
**Pull request by MaxGroenenboom**
Added basic support for indicators, updated the connector jar and changed the connection parameters.

User story: As a developer, I want my virtual human to be able to see its indicators, so it can work towards its goals.

### CODE REVIEW

File: src/DUWOagent/events.mod2g [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
 module events {
-	%%%%% Percept Processing %%%%%
+	%% When a percept is obtained, update the corresponding belief. %%
 	
-	
-	if percept(stakeholders(X)), bel(stakeholders(Y))  then  insert( stakeholders(X) ) + delete(stakeholders(Y)).
-	if percept(settings(X)), bel(settings(Y)) then insert(settings(X)) +delete(settings(Y)).
-	if percept(functions(X)), bel(functions(Y)) then insert(functions(X)) + delete(functions(Y)).
-	if percept(buildings(X)), bel(buildings(Y)) then insert(buildings(X))+delete(buildings(Y)).
+	if percept(stakeholders(X)), bel(stakeholders(Y))
```
Comment (klikooo) [inline src/DUWOagent/events.mod2g]: Maybe we should add some comments per percept

**Comment by MarcianoJ** [comment]
I agree with Rico about comments, besides that it looks good

**Comment by klikooo** [comment]
Okay looks good now!

**Comment by Denpeer** [comment]
tested and works

**Pull request merged**